es/lecciones/intro-a-google-maps-y-google-earth.md

```diff
@@ -0,0 +1,562 @@
+    hagas clic en "Cancelar" o "Aceptar") para poder ver el navegador de Google
+    Earth. El mapa que cargaste aparecerá sobre la imagen satelital de Google Earth
+    en el navegador.
+
+-   Hay marcadores en verde fosforescente en el medio y en los bordes del mapa
+    subido, que pueden ser utilizados para estirar, achicar y mover el mapa
+    para que se alinee correctamente con la imagen del satélite. Éste es un modo
+    sencillo de georeferenciar (mira [Georeferencing in QGIS 2.0][]). La imagen
+    de abajo muestra los pasos anteriores utilizando un viejo mapa de la ciudad
+    de Aultsville superpuesto a imágenes satelitales de Google de 2008 en el
+    cual se ven los restos de las calles y los cimientos de los edificios
+    en el río St. Lawrence (Aultsville fue uno de los "pueblos perdidos" que
+    fueron inundados por el proyecto de Vía Marítima y Energía de St. Lawrence).
+
+{% include figure.html filename="geo-es40.png" caption="Figura 40: Clic en la imagen para ver en tamaño completo" %}
+
+-   Volviendo a la ventana de "Nueva Superposición de Imágenes", fíjate que hay
+    una serie de opciones para seleccionar ("Descripción", "Ver", "Altitud",
+    "Actualizar", "Ubicación"). En esta instancia probablemente no necesites
+    preocuparte por ellas, pero quizás quieras agregar información bajo la
+    pestaña de Descripción.
+
+-   Cuando estés satisfecho con tu superposición, haz clic en "Aceptar" en
+    la esquina inferior derecha de la ventana de "Nueva Superposición de
+    Imágenes".
+
+-   Es importante que guardes tu trabajo. En "Archivo", en la barra del menú hay
+    dos opciones. Puedes guardar una copia de la imagen (Archivo -\> Guardar -\>
+    Guardar imagen...) que creaste en tu computadora en formato JPG o  guardar
+    la superposición de Google Earth para poder acceder a ella en
+    el futuro (Archivo -> Guardar -> Guardar en Mis Sitios). Esta segunda opción
+    genera un archivo KML.
+
+-   Para compartir archivos KML simplemente ubica el archivo que guardaste en tu
+    computadora y súbelo a tu sitio web, tu perfil de redes sociales o envíalo
+    como adjunto en un correo electrónico.
+
+**Ya aprendiste a utilizar Google Maps y Google Earth. ¡Asegúrate de guardar tu
+trabajo!**
+
+*Esta lección es parte de [Geospatial Historian][].*
+
+  [Google Maps Engine Lite]: https://mapsengine.google.com
+  [geo-es1]: /images/intro-a-google-maps-y-google-earth/geo-es1.png
+  [geo-es2]: /images/intro-a-google-maps-y-google-earth/geo-es2.png
+  [geo-es3]: /images/intro-a-google-maps-y-google-earth/geo-es3.png
+  [geo-es4]: /images/intro-a-google-maps-y-google-earth/geo-es4.png
+  [geo-es5]: /images/intro-a-google-maps-y-google-earth/geo-es5.png
+  [geo-es6]: /images/intro-a-google-maps-y-google-earth/geo-es6.png
+  [Archivo CSV del Suministro global de grasa de Reino Unido]: https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/Suministro global de grasa de Reino Unido 1894-1896.zip
```

Comment:
This ZIP file should be added to the `/assets` folder of this repository, and this link file should then be changed to point to `/assets/Suministro global de grasa de Reino Unido 1894-1896.zip`

Reply:
Travis IC says there an error with the .zip file. I uploaded it again after renaming the file Suministro_global_de_grasa_de_Reino_Unido_1894_1896.zip